website/blog/2023-02-01-technical-preview/index.mdx

```diff
@@ -0,0 +1,113 @@
+---
+slug: technical-preview
+title: v0.0.1 - Technical Preview
+authors: [ pmig, lw ]
+published: false
+tags: [ glasskube, release ]
+---
+
+import Install from '../../src/partials/_install.mdx';
+
+Glasskube is open-source on GitHub - Support us by leaving a star: [⭐ `glasskube/glasskube` ⭐](https://github.com/glasskube/glasskube/)
+
+
+*The aim of this post is to share our technical preview of how a cloud native package manager could work and what challenges need to be solved.*
+
+
+## The missing package manager for Kubernetes - Introducing Glasskube
```

Comment:
To be pedantic, you should use an em-dash here (see https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/types-of-dashes). 

I would maybe change the heading to 

Introducing Glasskube—The Missing Package Manager For Kubernetes